volume/local/local_unix.go

```diff
@@ -71,6 +68,44 @@ func (v *localVolume) mount() error {
 	if v.opts.MountDevice == "" {
 		return fmt.Errorf("missing device in volume options")
 	}
+	if strings.Contains(v.opts.MountOpts, "addr=") && v.opts.MountOpts != "" && v.opts.MountType != "" {
```

Comment:
No need for `strings.Contains`, a lookup to `if addr := GetAddress(v.opts.MountOpts); addr != "" { ... }` should suffice.